t/lib/t/MusicBrainz/Server/Controller/Release/Aliases.pm

```diff
@@ -26,6 +26,12 @@ test all => sub {
         },
         'releaseOf' => {
             'albumReleaseType' => 'http://schema.org/AlbumRelease',
+            'byArtist' => {
+                '@id' => 'https://musicbrainz.org/artist/4b585938-f271-45e2-b19a-91c634b5e396',
+                '@type' => ['Person', 'MusicGroup'],
+                'name' => 'Kate Bush',
+            },
+            'creditedTo' => 'Kate Bush',
```

Comment:
The release group AC is loaded unconditionally now, and happens to be outputted automatically with the release JSON-LD, which seems fine to me. But I could revert it to load the AC conditionally as before if desired.

Reply:
Unconditionally loading AC with release works for me, it is closely tightened to release info.